### PR TITLE
WCAG 2.1 for ListViewContents.tpl

### PR DESCRIPTION
**Pull request by rembiesa**

### CODE REVIEW

File: languages/en-US/_Base.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "Assets": "Assets",
     "OSSTimeControl": "Time Control",
     "Assigned To": "Assigned To",
+    "Empty Record": "Empty Record",
```
Comment (rskrzypczak) [inline languages/en-US/_Base.json:22]: remove

File: layouts/basic/modules/Vtiger/ListViewContents.tpl [hunk 2]
```diff
@@ -10,8 +10,8 @@
 ********************************************************************************/
 -->*}
 {strip}
-	<!-- tpl-Base-ListViewContents -->
 	{include file=\App\Layout::getTemplatePath('ListViewContentsTop.tpl', $MODULE_NAME)}
+	<!-- tpl-Base-ListViewContents -->
 	<table class="table tableBorderHeadBody listViewEntriesTable {$WIDTHTYPE} {if $VIEW_MODEL && !$VIEW_MODEL->isEmpty('entityState')}listView{$VIEW_MODEL->get('entityState')}{/if} js-fixed-thead" data-js="floatThead">
 		<thead>
 			<tr class="{if isset($CUSTOM_VIEWS) && $CUSTOM_VIEWS|@count gt 0}c-tab--border-active{/if} listViewHeaders">
```
Comment (rskrzypczak) [inline layouts/basic/modules/Vtiger/ListViewContents.tpl:17]: revert

File: layouts/basic/modules/Vtiger/ListViewContents.tpl [hunk 3]
```diff
@@ -111,7 +111,13 @@
 									{$LISTVIEW_ENTRY->getListViewDisplayValue($LISTVIEW_HEADER)}
 								</a>
 							{else}
-								{$LISTVIEW_ENTRY->getListViewDisplayValue($LISTVIEW_HEADER)}
+								<a href="">
```
Comment (rskrzypczak) [inline layouts/basic/modules/Vtiger/ListViewContents.tpl:114]: revert

**Review by rskrzypczak — CHANGES_REQUESTED** [review]

**Comment by rembiesa** [comment]
required changes in php

**Pull request closed without merging**